src/test/java/uk/gov/hmcts/reform/migration/query/ElasticSearchQueryTest.java

```diff
@@ -19,21 +19,51 @@ public void shouldReturnQuery() {
             .size(QUERY_SIZE)
             .build();
         String query = elasticSearchQuery.getQuery();
-        assertEquals("{\n"
-                         + "  \"query\": {\n"
-                         + "    \"match_all\": {}\n"
-                         + "  },\n"
-                         + "  \"_source\": [\n"
-                         + "    \"reference\"\n"
-                         + "  ],\n"
-                         + "  \"size\": 100,\n"
-                         + "  \"sort\": [\n"
-                         + "    {\n"
-                         + "      \"reference.keyword\": \"asc\"\n"
-                         + "    }\n"
-                         + "  ]\n"
-                         + "\n"
-                         + "}", query);
+        assertEquals("""
```

Comment:
A general comment here on the testing strategy - I'd suggest integration tests actually running against CCD that verify the migration behaves correctly rather than mocked tests like these where the tests are just a plastercast of the implementation.